internal/configs/version2/__snapshots__/templates_test.snap

```diff
@@ -101,6 +161,7 @@ server {
     listen 127.0.0.1:1234 ssl udp;
     listen [::1]:1234 ssl udp;
 
+    
```

Comment:
Are these newlines needed?

Reply:
They're not no. Updated in latest commit